kalliope/core/MainController.py

```diff
@@ -38,6 +38,13 @@ def __init__(self, brain=None):
 
         # create an order listener object. This last will the trigger callback before starting
         self.order_listener = OrderListener(self.analyse_order)
+
+        if self.settings.random_on_ready_answers is not None:
+            Say(message=self.settings.random_on_ready_answers)
+        elif self.settings.random_on_ready_sounds is not None:
+            random_sound_to_play = self._get_random_sound(self.settings.random_on_ready_sounds)
```

Comment:
Actually, as we can set a list of sounds, is it better to check all of them in settingsloader or should it be done when trying to actually access one file (in the elif) ?